src/main/java/org/prebid/server/auction/requestfactory/Ortb2ImplicitParametersResolver.java

```diff
@@ -84,6 +86,7 @@ public class Ortb2ImplicitParametersResolver {
     public Ortb2ImplicitParametersResolver(boolean shouldCacheOnlyWinningBids,
                                            String adServerCurrency,
                                            List<String> blacklistedApps,
+                                           ExtRequestPrebidServer serverInfo,
```

Comment:
Pls pass serverInfo properties, than create ExtRequestPrebidServer in class constructor